rb/lib/selenium/webdriver/bidi/browsing_context.rb

```diff
@@ -73,6 +73,35 @@ def get_tree(max_depth: nil)
           )
         end
 
+        def print_page(**options)
```

Comment:
I suggest we explicitly list out all available options in the method signature and then just refer to them

```ruby
def print_page(
  bottom: 1.0,
  top: 1.0,
  left: 1.0,
  right: 1.0,
  height: 27.94, 
  width: 21.59,
  page_ranages: [],
  scale: 1.0,
  shrink_to_fit: true
)
  print_result = @bidi.send_cmd(
    'browsingContext.print',
    context: @id,
    background: false,
    margin: {bottom: bottom, top: top, left: left, right: right},
    orientation: 'portrait',
    page: {height: height, width: width},
    pageRanges: page_ranges,
    scale: scale,
    shrinkToFit: shrink_to_fit
  )

  print_result['data']
end
```

Reply:
Should we create a basic PORO/Struct for options so we don't have so many optional parameters in this method? We can also use it for WebDriver classic print method.

Reply:
That would be nice actually too. Something like this should be enough:

```ruby
Margin = Struct.new(:bottom, :top, :left, :right, keyword_init: true) do 
   def initialize(bottom: 1.0, top: 1.0, left: 1.0, right: 1.0)
     super
   end
end
```

Reply:
Done